readme.md

```diff
@@ -237,6 +238,16 @@ programming languages with appropriate bindings for Ruby.
 - [lightgbm](https://github.com/ankane/lightgbm) &mdash;
   Ruby bindings for LightGBM.
   <sup>[[dep: LightGBM](#lightgbm)]</sup>
+  
+### Vector search
+- [pgvector](https://github.com/pgvector/pgvector) &mdash;
+  Open-source vector similarity search for Postgres
```

Comment:
It is a valuable link, but not a Ruby project. If required, add it to the section with dependencies (`[[dep: ...]()]`).